core/src/com/group66/game/helpers/DifficultyManager.java

```diff
@@ -0,0 +1,52 @@
+package com.group66.game.helpers;
+
+
+// TODO: Auto-generated Javadoc
+/**
+ * The Class DifficultyManager.
+ */
+public class DifficultyManager {
+    
+    /** The difficulty. */
+    public static String difficulty;
+    
+    /**
+     * Instantiates a new difficulty manager.
+     */
+    public DifficultyManager() {
+        difficulty = "easy";
```

Comment:
Yes will do than if we refactor. I'm just not quite acquainted with enums right now.